Rakefile

```diff
@@ -0,0 +1,12 @@
+require 'data_mapper'
+require './app/data_mapper_setup'
```

Comment:
You'll need to require the models as well for this to work, you could also include the server